### PR TITLE
Add alert banner for vendors

### PR DESCRIPTION
**Pull request by Riadabd**
## Ticket Number

DL-5667

## Ticket Description

This PR sets up a banner to notify vendors that their rights will be propagating to both `app-digitaal-loket` and 1app-worship-decisions-database`.

## How to Test

Build the frontend locally, access `http://localhost:4200` and check if the banner below shows up.

image

### CODE REVIEW

File: app/templates/vendors/index.hbs [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 <div class="au-o-body-container au-o-box au-o-layout">
   <div class="au-u-margin-top au-u-margin-bottom">
+    <AuAlert
+      @title="Ter info"
+      @skin="warning"
+      @icon="circle-info"
+      @size="small"
+      @closable={{false}}
+    >
+      <p>Wijzigingen van de rechten worden automatisch gepropageerd naar Loket en DatabankErediensten.</p>
```
Comment (Windvis) [inline app/templates/vendors/index.hbs:11]: The app seems to be using [ember-intl](https://github.com/ember-intl/ember-intl) so I would add a translation for this and use the `{{t}}` helper here as well (similar to line 1 in this file).

**Review by Windvis — COMMENTED** [review]
Code and screenshot looks good 👍 small remark.

File: app/templates/vendors/index.hbs (hunk 1 above)
Comment (Riadabd) [reply app/templates/vendors/index.hbs]: Nice! I updated the the code to pick up the english or dutch translations, as you suggested.

#### Dutch:

image

#### English:

image

**Review by Windvis — APPROVED** [review]

**Comment by Windvis** [comment]
Looks good 👍

**Pull request merged**